projects/cashmere/src/lib/table/cell-resizer.component.ts

```diff
@@ -0,0 +1,126 @@
+/* Based on an example posted at: https://github.com/angular/material2/issues/8312 */
+
+import {ChangeDetectionStrategy, Component, EventEmitter, HostBinding, HostListener, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {BindObservable} from './bind-observable/bind-observable';
+import {untilDestroyed} from '../util';
+import {fromEvent, Observable, Subscription} from 'rxjs';
+import {skip} from 'rxjs/operators';
+
+export class CellResizeEvent {
+    constructor(public width: number, public directionLeft: boolean) {}
+}
+
+@Component({
+    selector: 'hc-cell-resizer',
+    template: '<div class="hc-cell-resizer-left"></div><div class="hc-cell-resizer-right"></div>',
+    styleUrls: ['./cell-resizer.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
If anyone else is unfamiliar with `ChangeDetectionStrategy.OnPush`, this post helped me: https://blog.thoughtram.io/angular/2016/02/22/angular-2-change-detection-explained.html